### ISSUE TITLE
Shipkit fails when executed from Windows CMD

### ISSUE THREAD

**Issue by jjstreet** [mockito/shipkit#739]
Version: `2.0.28`

Stated simply, shipkit relies on exec tasks that try to execute commands in an OS-specific way.

Specifically, when I added shipkit to a project and performed the task `testRelease` from a windows command prompt, it fails with the following:

```
> gradle testRelease

> Configure project :
  Building version '0.0.1' (value loaded from 'version.properties' file).

> Task :testRelease FAILED
  Performing release in dry run, with cleanup:
    ./gradlew releaseNeeded performRelease releaseCleanUp -PdryRun


FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':testRelease'.
> A problem occurred starting process 'command './gradlew''
```

`./gradlew` is not how to execute the gradle wrapper from command. I suspect that there are more instances of this elsewhere. I just haven't run into them yet since I'm struggling just to try and perform a test release.

**Comment by jjstreet** [comment]
Might want to look at `GradleBuild` task types:

```
task cleanBuildPublish(type: GradleBuild) {
     tasks = ['clean', 'build', 'publish']
}
```

Might be a way to call tasks in a way gradle like way without resorting to executing the wrapper from a task (which seems kinda hacky?).

**Comment by jjstreet** [comment]
I looked at #745 and that will work. My actual problem was solved by #743 but calling using gradlew.bat should work on Windows .
